engine/engine.go

```diff
@@ -381,7 +381,7 @@ func (e *Engine) Start() error {
 	}
 
 	if e.Settings.EnablePortfolioManager {
-		if err = e.PortfolioManager.Start(); err != nil {
+		if err = e.PortfolioManager.Start(Bot.Settings.Verbose); err != nil {
```

Comment:
Instead of passing around Bot.Settings.Verbose here it could also just be set inside Start()

```
portfolio.Verbose= Bot.Settings.Verbose
```